google-cloud-dlp/.yardopts

```diff
@@ -0,0 +1,8 @@
+--no-private
+--title=DLP API
+--exclude lib/google/cloud/dlp/v2beta1
```

Comment:
If you exclude this path (`lib/google/cloud/dlp/v2beta1`), are you not also excluding the docs files in `lib/google/cloud/dlp/v2beta1/doc`?

I believe the correct path to exclude is: `lib/google/privacy/dlp/v2beta1`.